CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+# Contributing
+
+Thank you for your interest in contributing to the Tact tree-sitter grammar! We welcome contributions from everyone.
+
+## Quick Links
+- [Issues](https://github.com/tact-lang/tree-sitter-tact/issues)
+- [Pull Requests](https://github.com/tact-lang/tree-sitter-tact/pulls)
+- [Tree-sitter Documentation](https://tree-sitter.github.io/tree-sitter/)
+
+## Getting Started
+
+Before you begin:
+- Check if there's already an [open issue](https://github.com/tact-lang/tree-sitter-tact/issues) addressing your problem
+- Check out existing [pull requests](https://github.com/tact-lang/tree-sitter-tact/pulls) to see if someone else is working on similar changes
+
+## How to Contribute
+
+### Reporting Issues
+
+- Include code snippets and error messages
+- Include as much detail as possible
+- Include steps to reproduce the issue
+- Include the version of tree-sitter you're using
+- Use code blocks for error messages and logs
+
+### Making Changes
+
+1. Fork the repository
+2. Create a new branch for your changes
+3. Make your changes
+4. Write or update tests as needed
+5. Ensure all tests pass
+6. Format the code using `npm run fmt`
+7. Submit a pull request
+
+### Pull Request Process
+
+1. Update the README.md if needed
+2. Follow the existing code style
+3. Include a clear description of your changes
+4. Link any related issues using keywords (e.g., "Fixes #123")
+5. Wait for review from maintainers
```

Comment:
I'd probably removed this altogether or moved it somewhere to highlight the dev setup and the overall structure of the project which follows below